README.md

```diff
@@ -1,8 +1,40 @@
 # ☄️ Effector
 
-Reactive state manager
+Reactive and effective state manager.
```

Comment:
effective - in not a technical characteristic, but evaluative judgement, I guess. Does it seem good for title?
--

Reply:
I think yes🤔